ACPs/131-cancun-eips/README.md

```diff
@@ -0,0 +1,61 @@
+```text
```

Comment:
Could you update the PR to use the newly introduced table format for the header?
```
| ACP | PR Number |
| :--- | :--- |
| **Title** | ACP title |
| **Author(s)** | A list of the author's name(s) and optionally contact info: FirstName LastName ([@GitHubUsername](https://github.com/Username) or [email@address.com](mailto:email@address.com)) |
| **Status** | Proposed, Implementable, Activated, Stale ([Discussion](POPULATED BY MAINTAINER, DO NOT SET)) |
| **Track** | Standards, Best Practices, Meta, Subnet |
| **Replaces (\*optional)** | [ACP-XX](../XXX/README.md) |
| **Superseded-By (\*optional)** | [ACP-XX](../XXX/README.md) |
```